profile/repositories-list.md

```diff
@@ -0,0 +1,147 @@
+| [SIOPv2 & OpenID4VP protocols (Android)](https://github.com/eu-digital-identity-wallet/eudi-lib-jvm-siop-openid4vp-kt) | Implementation of SIOPv2 and OpenID4VP (draft 24) protocols (wallet's role) in Kotlin. | 
+| [SD-JWT (Android)](https://github.com/eu-digital-identity-wallet/eudi-lib-jvm-sdjwt-kt)                        | SD-JWT library for creating and verifying SD-JWT in JVM Kotlin.             | 
+
+### Ιssuing iOS Libraries
+
+| Repository          | Description                                                                   | 
+|---------------|-------------------------------------------------------------------------------|
+| [OpenId4VCI (iOS)](https://github.com/eu-digital-identity-wallet/eudi-lib-ios-openid4vci-swift) | Implementation of credential management supporting the OpenId4VCI (draft 15) protocol. | 
+
+### Ιssuing Android Libraries
+
+| Repository          | Description                                                                   | 
+|---------------|-------------------------------------------------------------------------------|
+| [OpenId4VCI (Android)](https://github.com/eu-digital-identity-wallet/eudi-lib-jvm-openid4vci-kt)      | Implementation of credential management supporting the OpenId4VCI (draft 15) protocol.|
+
+### Wallet Data Storage and Cryptographic Management iOS Libraries
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [mDoc Document Storage (iOS)](https://github.com/eu-digital-identity-wallet/eudi-lib-ios-wallet-storage) |  Storage for keys and wallet documents |
+
+### Wallet Data Storage and Cryptographic Management Android Libraries
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [mDoc Document Storage (Android)](https://github.com/eu-digital-identity-wallet/eudi-lib-android-wallet-document-manager)    | This library provides a set of classes to manage documents and their cryptographic keys in an EUDI Android Wallet. |
+
+
+### Revocation Libraries
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [Token Status List (Kotlin)](https://github.com/eu-digital-identity-wallet/eudi-lib-kmp-statium) |  Statium is a Kotlin multiplatform library supporting JVM and Android platforms. It implements the Token Status List Specification draft 10, and allows callers to check the status of a "Referenced Token" as defined in the specification, effectively enabling applications to verify if tokens are valid, revoked, or in other states. |
```

Comment:
Ensure the spec draft number is accurate/consistent (we cite 'draft 10' here; elsewhere 'draft 12').